libs/vertexai/tests/unit_tests/test_vectorstores.py

```diff
@@ -76,3 +86,74 @@ def test_add_texts_with_custom_ids(mocker):
         VectorStore.add_texts(texts=texts, ids=["Id1"])
     with pytest.raises(ValueError):
         VectorStore.add_texts(texts=texts, ids=["Id1", "Id2", "Id2"])
+
+
+def test_add_texts_with_embeddings():
```

Comment:
it's great we have a unit test! do you think it's worth adding an integration test too, please?

Reply:
added 2 integration tests:
- `test_vector_store_hybrid_search`
- `test_add_texts_with_embeddings`